lib/logstash/codecs/cef.rb

```diff
@@ -604,9 +604,9 @@ def desanitize_extension_val(value)
   end
 
   def normalize_timestamp(value, device_timezone_name)
-    value = @timestamp_normalzer.normalize(value, device_timezone_name).iso8601(9)
+    normalized = @timestamp_normalzer.normalize(value, device_timezone_name).iso8601(9)
```

Comment:
I know it is out of scope for this PR but that typo is making my teeth itch... 😆